src/graphic/helper/dashStyle.ts

```diff
@@ -0,0 +1,14 @@
+import { isArray, isNumber } from '../../core/util';
+
+export function normalizeLineDash(lineType: any, lineWidth?: number): number[] | false {
+    if (!lineType || lineType === 'solid' || (lineWidth != null && lineWidth <= 0)) {
+        return null;
```

Comment:
```ts
(lineWidth != null && lineWidth <= 0))
```
can be simplified to
```ts
!(lineWidth > 0)
```

It will also reject other potential invalid NaN values.

Sorry for being very critical in this review. Because it's at the very bottom of code that may affect a lot.